src/components/SplitButton/SplitButtonMenu/__tests__/splitButtonMenu-snapshot-tests.jest.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import SplitButtonMenu from "../SplitButtonMenu";
+import MenuItem from "../../../Menu/MenuItem/MenuItem";
+import { mockRequestAnimationFrame, restoreRequestAnimationFrameMock } from "./test-utils";
+
+const SplitButtonMenuWithItems = (
+  <SplitButtonMenu id="menu">
+    <MenuItem title="Test 1" />
+    <MenuItem title="Test 2" />
+  </SplitButtonMenu>
+);
+
+describe("SplitButtonMenu", () => {
+  beforeEach(() => {
+    mockRequestAnimationFrame();
```

Comment:
what is this and what the purpose of this?

Reply:
remember we spoke about adding requestAnimationFrame for the focus in the component?
Then we need to mock it in the tests to be able to test the focus (we need to wait for the next frame, after the focus is being done)